modules/apps/user-associated-data/user-associated-data-web/src/main/java/com/liferay/user/associated/data/web/internal/configuration/AnonymousUserConfigurationRetriever.java

```diff
@@ -30,7 +30,7 @@
 @Component(service = AnonymousUserConfigurationRetriever.class)
 public class AnonymousUserConfigurationRetriever {
 
-	public Optional<Configuration> getOptional(long companyId)
+	public Configuration getOptional(long companyId)
```

Comment:
@LisztLiszt When modifying such methods, as well as their usages, please rename them directly.

So, when you change this method from returning `Optional<Configuration>` to `Configuration`, you should rename the method to `get` in the same commit, and rename the variable that receives the return value in the usage from `configurationOptional` to `configuration` in your "apply usage" commit.